listings/views.py

```diff
@@ -85,3 +90,42 @@ class ReviewView(HelpOthersMetaDataMixin, TemplateView):
     Page to notify the user that the resource will be reviewed.
     """
     template_name = 'listings/resources/review.html'
+
+
+class LikeView(TemplateView, View):
```

Comment:
- TemplateVIew extends View, so no need to specify both.
- If you use LoginRequiredMixin it will redirect users to login page and then back to this.